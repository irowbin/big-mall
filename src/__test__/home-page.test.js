import { shallow } from 'enzyme'
import HomePage from '../pages/homepage/homepage.component'

describe('Homepage component', () => {
  it('should cmatch snapshot', () => {
    expect(shallow(<HomePage />)).toMatchSnapshot()
  })
})
