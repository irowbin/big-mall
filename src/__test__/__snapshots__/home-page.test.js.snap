// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage component should cmatch snapshot 1`] = `ShallowWrapper {}`;
